ui/client/containers/TableSelect.js

```diff
@@ -0,0 +1,51 @@
+import React from "react"
```

Comment:
I think this is wrong space (/client/containers/) for this component..

Reply:
Done